src/cljs/commiteth/update_address.cljs

```diff
@@ -3,14 +3,16 @@
             [commiteth.common :refer [input dropdown]]
             [reagent.core :as r]
             [reagent.crypt :as crypt]
+            [clojure.string :as str]
             [cljs-web3.eth :as web3-eth]))
 
 
 (defn update-address-page []
   (let [db (rf/subscribe [:db])
         user (rf/subscribe [:user])
         updating-address (rf/subscribe [:get-in [:updating-address]])
-        address (r/atom @(rf/subscribe [:get-in [:user :address]]))]
+        address (r/atom (some-> @(rf/subscribe [:get-in [:user :address]]) 
+                                str/lower-case))]
     (fn []
       (let [web3 (:web3 @db)
             web3-accounts (when web3
```

Comment:
I know it already was like that, but how about taking the web3 accounts from a re-frame subscription?

Reply:
I suggest addressing it in scope of #242 .